Code/Source/RobotImporter/RobotImporterWidget.cpp

```diff
@@ -63,20 +76,23 @@ namespace ROS2
                     ReportError("User cancelled");
                     return;
                 }
-                RobotImporter::Import(
-                    { urdfPath.value(), prefabPath.value() },
-                    [this](const AZStd::string& message)
-                    {
-                        ReportInfo(message);
-                    },
-                    [this](const AZStd::string& message)
-                    {
-                        ReportError(message);
-                    });
+                m_robotImporter.Import({ urdfPath.value(), prefabPath.value() });
+                m_selectFileButton->setEnabled(false);
+                m_importerUpdateTimer->start(500);
```

Comment:
Comment on this please - are we prohibiting user from running this twice by mistake, or something else? Why 500 - and perhaps using a named constant or comment to make clear it is miliseconds?